strings/steve.json

```
@@ -15,6 +19,11 @@
         "We should store bank records in a webscale datastore, like /dev/null.",
         "You ever wonder if /dev/null supports sharding?",
         "Do you know who ate all the donuts?",
-        "What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?"
+        "What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?",
+        "Bruh what if we took a locker, some random trash and made a mech out of it?",
+        "Ok here me out, what if we make illegal, not illegal so sec stops arresting us for having it.",
+        "Crazy idea, ok ok ok, rather then having monkeys to test on, what if we only used apes.",
```

Than not then
Sentence is drawn out by constant use of commas and connections. Intro (the "crazy idea" part of the sentence) should be shorter.
Question mark not period
```suggestion
        "I have a crazy idea, guys. Rather than having monkeys to test on, what if we only used apes?",
```